test/fixtures/v3/did-revocation-store-signed-not-revoked.json

```diff
@@ -12,13 +12,10 @@
   "validFrom": "2010-01-01T19:23:24Z",
   "issuer": {
     "id": "https://example.com",
+    "type": "OpenAttestationIssuer",
```

Comment:
i really need to read up more on json-ld, did you read the spec to understand the general concepts of it?

Reply:
No it's because most of the files were not v3 valid due to the latest changes of context. In oa-verify we don't wrap document so we can't really see whether a file is valid or not

I added location for this MR because it was missing in the context.

I read the doc (not fully) but I didn't find it helpful. Fak3 helped me to understand some concepts (how to apply scope with types)

Reply:
ah ic.

👌

okok, then i'll read up the discussion you had with Fak3 that is somewhere around (plus the spec if im bored).